Add projects autostart switch (#63)
docksal/service-vhost-proxy: README.md

## Advanced proxy configuration

These advanced settings can be used in CI sandbox environments and help keep the resource usage down by stopping 

> Docksal project containers after a period of inactivity. Projects are automatically restarting upon a new HTTP request (depending from PROJECT_AUTOSTART. see below).

Reviewer: ```suggestion
Docksal project containers after a period of inactivity. Projects are automatically restarted upon a new HTTP request (unless `PROJECT_AUTOSTART` is set to `0`, see below.).
```

Note: permanent projects will still be put into hibernation according to `PROJECT_INACTIVITY_TIMEOUT`.

`PROJECT_AUTOSTART_DISABLE`

> Setting this variable to `1` will disable autostart projects by viziting project url. This option is inactive by default (set to `0`).

Reviewer 2: visiting
